ml-agents/mlagents/trainers/components/reward_signals/gail/signal.py

```diff
@@ -82,7 +82,7 @@ def evaluate_batch(self, mini_batch: Dict[str, np.array]) -> RewardSignalResult:
         if self.policy.use_continuous_act:
             feed_dict[self.policy.selected_actions] = mini_batch["actions"]
         else:
-            feed_dict[self.policy.action_holder] = mini_batch["actions"]
+            feed_dict[self.policy.output] = mini_batch["actions"]
```

Comment:
feels like we are overloading `self.policy.output` to act as both a placeholder self.policy.action_holder and also the tensor that represents the tensorflow function that composes the policy (since we backprop through it when reparameterizing).